Update release notes for TensorFlow 2.5.0

tensorflow-jenkins opened the pull request:
This PR is intentionally incomplete. One of the Release Owners for 2.5.0
needs to fill in the internal release notes for this version before the PR gets
submitted. Click on the :pencil2: icon in the header for `RELEASE.md` under
"Files Changed" above.

tensorflow-jenkins committed:
Insert release notes place-fill

geetachavan1 committed:
Update Release Notes

mattdangerw committed:
Update release notes with Keras preprocessing changes

mihaimaruseac committed:
Update RELEASE.md

goldiegadde committed:
Update RELEASE.md

geetachavan1 committed:
Update RELEASE.md

mihaimaruseac committed:
Update RELEASE.md

duncanriach commented:
I don't seem to be authorized to edit this PR. Please will someone add the changes from [closed PR 48581](https://github.com/tensorflow/tensorflow/pull/48581/files), incorporating @sanjoy's spelling correction?

geetachavan1 commented:
Sure I will take care of it

On Tue, Apr 20, 2021 at 2:30 PM Duncan Riach ***@***.***>
wrote:

> I don't seem to be authorized to edit this PR. Please will someone add the
> changes from closed PR 48581
> <https://github.com/tensorflow/tensorflow/pull/48581/files>,
> incorporating @sanjoy <https://github.com/sanjoy>'s spelling correction?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/48090#issuecomment-823612207>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMWX6TKVIQJ4SDQX3M7DHTLTJXW65ANCNFSM4Z2NTT5A>
> .
>

geetachavan1 committed:
Adding Deterministic Op Functionality 

Adding Deterministic Op Functionality  https://github.com/tensorflow/tensorflow/pull/48581

penpornk committed:
Update RELEASE.md

Change wording about PluggableDevice to avoid confusion.

mihaimaruseac committed:
Update RELEASE.md
